packages/manager/modules/pci/src/projects/project/failover-ips/order/legacy/order.controller.js

```diff
@@ -25,28 +23,28 @@ export default class FailoverIpController {
 
 
   onInstanceChange(instance) {
-    if (this.ip.region) {
-      this.checkCompatibility(instance, this.ip.region);
-    }
-
-    const continent = find(this.regions, { name: instance.region }).continentCode;
-
-    this.REGIONS = REGIONS[continent].map(region => ({
+    const countries = find(this.regions, { name: instance.region }).ipCountries;
```

Comment:
You can use destructuring assignment for instance:

```suggestion
    const { ipCountries: countries } = find(this.regions, { name: instance.region });
```